cpp/oneapi/dal/algo/pca/test/fixture.hpp

```diff
@@ -212,8 +212,18 @@ class pca_test : public te::crtp_algo_fixture<TestType, Derived> {
     }
 
     void check_infer_result(const pca::descriptor<Float, Method>& desc,
-                            const te::dataframe& data,
-                            const pca::infer_result<>& result) {}
+                            const pca::model<>& model,
+                            const table& data,
+                            const pca::infer_result<>& result) {
+        const auto eigenvectors = model.get_eigenvectors();
```

Comment:
Please add a TODO mark to extend tests via gold data and tests for whitening/scaling. For future work